plugin/pkg/admission/persistentvolume/label/admission_test.go

```diff
@@ -67,6 +68,13 @@ func (c *mockVolumes) DisksAreAttached(nodeDisks map[types.NodeName][]aws.Kubern
 	return nil, fmt.Errorf("not implemented")
 }
 
+func (c *mockVolumes) ResizeDisk(
```

Comment:
Why is this change in the last commit? Should it be in an earlier one?

Reply:
The earlier commits contain code from previous PR - https://github.com/kubernetes/kubernetes/pull/55815/files . The last commit introduces Disk resizing for EBS volumes and that is why this change is only there in the last commit.